### PR TITLE
Add clarity initialization in the MainActivity.java

### PR DESCRIPTION
**Pull request by hydraxman**
## Description

We want to leverage clarity to understand the usage data of Hydra Lab client. This can enable data dashboard, error diagnostic, etc.

A related update to reflect the change: https://github.com/microsoft/HydraLab/pull/637/files

## Pull Request Checklist

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Code compiles correctly with all tests are passed.
- [x] I've read the [contributing guide](https://github.com/microsoft/HydraLab/blob/main/CONTRIBUTING.md#making-changes-to-the-code) and followed the recommended practices.
- [x] [Wikis](https://github.com/microsoft/HydraLab/wiki) or [README](https://github.com/microsoft/HydraLab/blob/main/README.md) have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?
*If this introduces a breaking change for Hydra Lab users, please describe the impact and migration path.*

- [x] Yes
- [ ] No

## How you tested it
*Please make sure the change is tested, you can test it by adding UTs, do local test and share the screenshots, etc.*

Build and run the app. This integration practice comply with [android-sdk installation](https://learn.microsoft.com/en-us/clarity/mobile-sdk/android-sdk).

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Technical design
- [ ] Build related changes
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Code style update (formatting, renaming) or Documentation content changes
- [ ] Other (please describe): 

### Feature UI screenshots or Technical design diagrams
*If this is a relatively large or complex change, kick it off by drawing the tech design with PlantUML and explaining why you chose the solution you did and what alternatives you considered, etc...*

### CODE REVIEW

**Comment by AC-Phoenix** [comment]
We also need to add dependency in build.gradle: https://learn.microsoft.com/en-us/clarity/mobile-sdk/android-sdk

**Comment by hydraxman** [comment]
> We also need to add dependency in build.gradle: https://learn.microsoft.com/en-us/clarity/mobile-sdk/android-sdk

Sure, it's added in another PR. Which is why you don't see the change in this one.

File: agent/build.gradle [hunk 1]
```diff
@@ -64,4 +64,6 @@ dependencies {
 
     implementation 'io.appium:java-client:8.0.0'
     implementation 'org.seleniumhq.selenium:selenium-java:4.1.4'
+
+    implementation 'com.microsoft.clarity:clarity:2.1.1'
```
Comment (hydraxman) [inline agent/build.gradle:68]:
```suggestion
```

File: agent/build.gradle [hunk 2]
```diff
@@ -64,4 +64,5 @@ dependencies {
 
     implementation 'io.appium:java-client:8.0.0'
     implementation 'org.seleniumhq.selenium:selenium-java:4.1.4'
+
```
Comment (hydraxman) [inline agent/build.gradle:67]:
```suggestion
```

**Review by zhou9584 — APPROVED** [review]

**Pull request merged**